archivist/composite.py

```diff
@@ -0,0 +1,99 @@
+"""Composite interface
+
+   Access to various utility methods that access various endpoints.
+
+   The user is not expected to use this class directly. It is an attribute of the
+   :class:`Archivist` class.
+
+   For example instantiate an Archivist instance and execute the methods of the class:
+
+   .. code-block:: python
+
+      with open(".auth_token", mode="r", encoding="utf-8") as tokenfile:
+          authtoken = tokenfile.read().strip()
+
+      # Initialize connection to Archivist
+      arch = Archivist(
+          "https://app.rkvst.io",
+          authtoken,
+      )
+      asset = arch.composite.......(...)
+
+"""
+
+from collections import Counter
+from logging import getLogger
+
+# pylint:disable=unused-import      # To prevent cyclical import errors forward referencing is used
+# pylint:disable=cyclic-import      # but pylint doesn't understand this feature
+from . import archivist as type_helper
+
+
+LOGGER = getLogger(__name__)
+
+
+class _CompositeClient:
```

Comment:
I don't think we really need this. The double-check method and duplicate finder were there from the very early days when it was possible to pick up double records from blockchain caches and things. That doesn't happen anymore, so the YAML that just counts the 3 things is enough.